src/main/webapp/script.js

```diff
@@ -696,9 +719,46 @@ function searchToken(cy, query) {
       target = target.add(node);
     }
   });
+
   if (target.length > 0) {
-    return target;
+    document.getElementById('num-selected').innerText = "out of " + target.length;
+    document.getElementById('highlight-number').min = 1;
+    document.getElementById('highlight-number').value = 1;
+    document.getElementById('highlight-number').max = target.length;
+    document.getElementById('highlight-number').disabled = false;
+    document.getElementById('highlight-number').onchange = function () {
+      updateHighlightedToken(cy, target, document.getElementById('highlight-number').value - 1);
+    };
+    document.getElementById('prevnode').onclick = function () {
```

Comment:
nice is this based on my FR during the daily sync? I think the buttons should probably only be displayed if the target.length > 1 because there's no point having such buttons when there is only one node to highlight.

Reply:
yes it is, should they be removed completely (display: none) or just disabled? also should they be removed/disabled when token search is not being used? ie. node search or filtering was just executed

Reply:
I'm working on the disabled part. I'll push in a few.

Reply:
i think it should be removed? but either could work what do you think?